### ISSUE TITLE
Access denied

### ISSUE THREAD

**Issue by bystefu** [Athlon1600/youtube-downloader#32]
I get 

`Access to r4---sn-gqn-jawe.googlevideo.com was denied You don't have authorization to view this page.
HTTP ERROR 403`

Also i tried with file_get_contents, or file_put_contents to see if i must to use same ip, but nothing...

**Comment by InvoxiPlayGames** [comment]
I get the same error above, when viewing content most likely to be copyrighted (music videos, for example), even when content is proxied from the server IP. Most likely a signature issue as YouTube change it very frequently.

**Comment by shojaeix** [comment]
same issue

**Comment by lucaasleaal** [comment]
#20 

looks like it is the same problem. Also happening to me. I'm considering changing my project to exec youtube-dl right now.

**Comment by snnyk** [comment]
same error at my projects any idea or alternative ?

**Comment by chmas123chmas123** [comment]
U can use php stream to solve that

Filmyou.ir  is a good example of php stream to bypass this problem

**Comment by jammin88** [comment]
Same issue here

**Comment by sajjad-ser** [comment]
> U can use php stream to solve that
> 
> http://filmyou.ir is a good example of php stream to bypass this problem

can you share the source code?

**Comment by PhamMinhKha** [comment]
same issure. video get status 403 
![Screenshot from 2020-01-27 20-11-52](https://user-images.githubusercontent.com/12944689/73177323-51d7d380-4141-11ea-86b1-1b7b54d8ea2d.png)

**Comment by hoofdtelefoon** [comment]
Same for me

**Comment by Athlon1600** [comment]
are you using the latest version? because this still works just fine on our demo app here:
https://github.com/Athlon1600/youtube-downloader#demo-app

the other solution would be to just "stream" whatever video you get from `youtube-downloader` via your own server. see this:  
https://github.com/Athlon1600/youtube-downloader/blob/master/public/stream.php

**Comment by hoofdtelefoon** [comment]
Yes I’m using the latest version but the problems seems to be on my server, I have some problems with my SSL certificates.
Still trying to figure it out. Thanks for your help anyways.

Van: Athlon1600<mailto:notifications@github.com>
Verzonden: zaterdag 9 mei 2020 03:40
Aan: Athlon1600/youtube-downloader<mailto:youtube-downloader@noreply.github.com>
CC: hoofdtelefoon<mailto:peerdenhoed@hotmail.nl>; Comment<mailto:comment@noreply.github.com>
Onderwerp: Re: [Athlon1600/youtube-downloader] Access denied (#32)


are you using the latest version? because this still works just fine on our demo app here:
https://github.com/Athlon1600/youtube-downloader#demo-app

the other solution would be to just "stream" whatever video you get from youtube-downloader via your own server. see this:
https://github.com/Athlon1600/youtube-downloader/blob/master/public/stream.php

**Comment by socialAPIS** [comment]
You can use my api, find it in my repo, and as i discovered you dont have a download rate, just ask for new key

**Comment by mrxx** [comment]
Same issue here

**Comment by RemiC2care** [comment]
Same issue Here,

It work when i'm on localhost, but get 403 on some video when live.
